crates/argmin/Cargo.toml

```diff
@@ -43,7 +43,7 @@ argmin-checkpointing-file = { path = "../argmin-checkpointing-file" }
 [features]
 default = []
 wasm-bindgen = ["getrandom/js"]
-serde1 = ["serde", "rand_xoshiro/serde1"]
```

Comment:
Why is this feature removed instead of changed to `rand_xoshiro/serde` as mentioned in the [changelog](https://github.com/rust-random/rngs/blob/master/rand_xoshiro/CHANGELOG.md)?